src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/ReplyCreator.java

```diff
@@ -1,5 +1,7 @@
 package com.wikia.webdriver.elements.mercury.components.discussions.common;
 
+import java.net.URL;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.net.URL'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)